ccc/github.py

```diff
@@ -63,6 +66,46 @@ def github_api_ctor(
         raise ValueError('failed to parse url: ' + str(github_url))
 
     session = github3.session.GitHubSession()
+    original_request = session.request
+
+    short = False
```

Comment:
consider naming this `skip_logging` or even `skip_logging_to_els` or something.

Reply:
+1
as an alternative, one might also consider `shortcut_logging` (as for naming)

however, if we are persistently shortcutting logging (which is a valid approach), why not rather patch-back to the original method, thus saving a little overhead?

also, in addition to logging a warning, it would be nice to also state the future requests in same session will not even be attempted to be logged

Reply:
do agree, patching back instead of shortcutting is better 👍